src/components/apphost.js

```diff
@@ -453,7 +453,8 @@ define(["appSettings", "browser", "events", "htmlMediaHelper", "webSettings"], f
 
     if (doc) {
         doc.addEventListener(visibilityChange, function () {
-            if (document[visibilityState]) {
+            /* eslint-disable-next-line compat/compat */
+            if (document.visibilityState) {
```

Comment:
`document[visibilityState]` was use `visibilityState` variable that set above. `document.visibilityState` is supposed be polyfilled.

Reply:
Hm, this entire thing seems a little screwy and convoluted.

It should work in a different way, probably. I'll try refactoring it all tonight.

Reply:
Should be better now :)